This patch backports field types that were added after MySQL 5.5,
but are technically still compatible to the old client libs.

To apply:
	cd mysql-source-root
	patch -p1 < this-file

(patch created with `diff -Naur`)

--- mysql-5.5.56-dist/include/mysql_com.h	2017-04-27 09:12:30.000000000 +0200
+++ mysql-5.5.56/include/mysql_com.h	2017-05-21 01:46:44.000000000 +0200
@@ -349,7 +349,11 @@
 			MYSQL_TYPE_DATETIME, MYSQL_TYPE_YEAR,
 			MYSQL_TYPE_NEWDATE, MYSQL_TYPE_VARCHAR,
 			MYSQL_TYPE_BIT,
-                        MYSQL_TYPE_NEWDECIMAL=246,
+			MYSQL_TYPE_TIMESTAMP2,
+			MYSQL_TYPE_DATETIME2,
+			MYSQL_TYPE_TIME2,
+			MYSQL_TYPE_JSON=245,
+			MYSQL_TYPE_NEWDECIMAL=246,
 			MYSQL_TYPE_ENUM=247,
 			MYSQL_TYPE_SET=248,
 			MYSQL_TYPE_TINY_BLOB=249,
